src/Decoder/SmileDecoder.php

```diff
@@ -127,14 +127,14 @@ private function decodeKey(int $byte): DecodingResult
     {
         return match (true) {
             $byte < Bytes::THRESHOLD_SIMPLE_LITERALS => new DecodingResult(null, true), // 0 > 31 are reserved
-            Bytes::LITERAL_EMPTY_STRING === $byte => new DecodingResult('""'),
+            Bytes::LITERAL_EMPTY_STRING === $byte => new DecodingResult(''),
             $byte < Bytes::THRESHOLD_LONG_SHARED_KEY => new DecodingResult(null, true), // 33 > 37 are reserved
-            $byte < Bytes::KEY_LONG_KEY_NAME => new DecodingResult(null, true), // TODO: implement long shared key references
-            Bytes::KEY_LONG_KEY_NAME === $byte => new DecodingResult(null, true), // TODO: implement long key name
+            // TODO: Long Key Name don't seem to exist. Update on the enum needed.
```

Comment:
TODO

Reply:
I intend to clear them all in a future PR, this is just to prevent me from forgetting it, we have a lot of TODO currently